core/App/components/misc/CredentialCard.tsx

```diff
@@ -43,19 +41,31 @@ const CredentialCard: React.FC<CredentialCardProps> = ({
     if (proof) {
```

Comment:
Because these are all independent conditionals with a return do you think it would read better is separate if statements and use the "early return" pattern?

```JavaScript
if (a) {
   return 1 + 1
}

if (b) {
   return 2 + 1
}

if (c) {
   return 3 + 1
}

return 5 + 5
}
```

General I find if / else better where the conditional is similar like

```
if (a <= 5)
return 'cat'
else if (a >5 && <= 10)
return 'dog'
else // for other values of a
return 'unknown'
```